kitsune/community/cron.py

```diff
@@ -0,0 +1,80 @@
+import cronjobs
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+from django.db.models import F, Q
+
+from tower import ugettext as _
+
+from kitsune.questions.models import Answer
+from kitsune.sumo.email_utils import make_mail, safe_translation, send_messages
+from kitsune.users.models import Profile
+from kitsune.wiki.models import Revision
+
+
+@cronjobs.register
+def send_welcome_emails():
+    """Send a welcome email to first time contributors.
+
+    Anyone who has made a contribution more than 24 hours ago and has not
+    already gotten a welcome email should get a welcome email.
+    """
+
+    wait_period = datetime.now() - timedelta(hours=24)
+    messages = []
+    context = {
+        'host': Site.objects.get_current().domain,
+    }
+
+    # Answers
+
+    answer_filter = Q(created__lte=wait_period)
+    answer_filter &= ~Q(question__creator=F('creator'))
+    answer_filter &= Q(creator__profile__first_answer_email_sent=False)
+
+    answer_recipient_ids = set(
+        Answer.objects
+        .filter(answer_filter)
+        .values_list('creator', flat=True))
+
+    @safe_translation
+    def _make_answer_email(locale, to):
+        return make_mail(subject=_('Thank you for your contribution to Mozilla Support!'),
```

Comment:
This subject line has now been approved. We may change it in the future, but this can merge with it.